tortoisebeacon/handlers.go

```diff
@@ -75,7 +75,8 @@ func (tb *TortoiseBeacon) handleProposalMessage(ctx context.Context, m ProposalM
 		tb.Log.With().Warning("Received malformed proposal message: VRF is not verified",
 			log.String("sender", m.MinerID.String()),
 			log.String("sender_short", m.MinerID.ShortString()))
-		return nil
+
+		return ErrMalformedProposal
```

Comment:
this introduces another edge case, document that maybe additional test is needed for this one